onnx/backend/test/case/node/scatter.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Scatter(Base):
+
+    @staticmethod
+    def export_scatter_without_axis():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Scatter',
+            inputs=['data', 'indices', 'updates'],
+            outputs=['y'],
+        )
+        data = np.zeros((3, 3))
+        indices = np.array([[1, 0, 2], [0, 2, 1]])
```

Comment:
please specify the dtype as int64, otherwise, we may generate different type of data on 32bit platform.

Reply:
sure, resolved. good point!